Bottom: d9257ee03dd1ab196d8b1cf5973fb1009e9ef6bb
Top:    51f07d92a2b5d0906dfa40d05a1c21a80cdf1b26
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 17:30:24 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 22d981c..a16af92 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1358,10 +1358,6 @@ DrawMask (BoxType * screen)
       gui->set_color (Output.pmGC, "erase");
     }
 
-  gui->use_mask (HID_MASK_BEFORE);
-  gui->set_color (out->fgGC, PCB->MaskColor);
-  gui->fill_rect (out->fgGC, 0, 0, PCB->MaxWidth, PCB->MaxHeight);
-
   gui->use_mask (HID_MASK_CLEAR);
   r_search (PCB->Data->pin_tree, screen, NULL, clearPin_callback_solid, &info);
   r_search (PCB->Data->via_tree, screen, NULL, clearPin_callback_solid, &info);
